src/components/breadcrumbs/breadcrumb.tsx

```diff
@@ -48,13 +48,25 @@ export type EuiBreadcrumbProps = Omit<
      */
     truncate?: boolean;
     /**
-     * Accepts any EuiLink `color` when rendered as one (has `href` or `onClick`)
+     * Accepts any EuiLink `color` when rendered as one (has `href`, `onClick`, or `popoverContent`)
      */
     color?: EuiLinkColor;
     /**
      * Override the existing `aria-current` which defaults to `page` for the last breadcrumb
      */
     'aria-current'?: AriaAttributes['aria-current'];
+    /**
+     * Creates a breadcrumb that toggles a
```

Comment:
I read this comment a couple of times and wondered if it would be more clear if we named the UI that'll appear. Maybe I'm being too pedantic, feel free to use or pass.

```suggestion
     * Creates a breadcrumb that toggles a popover dialog
```

Reply:
lmao I definitely didn't just forgot to finish typing an entire sentence there 😂 thanks for catching that Trevor!